vela/client.go

```diff
@@ -406,6 +406,42 @@ func (c *Client) Call(method, u string, body interface{}, v interface{}) (*Respo
 	return resp, err
 }
 
+// CallWithHeaders is a combined function for Client.NewRequest and Client.Do.
+//
+// Most API methods are quite the same.
+// Get the URL, apply options, make a request, and get the response.
+// Without adding special headers or something.
+// To avoid a big amount of code duplication you can Client.Call.
+//
+// method is the HTTP method you want to call.
+// u is the URL you want to call.
+// body is the HTTP body.
+// v is the HTTP response.
+// headers is a map of HTTP headers
+//
+// For more information read https://github.com/google/go-github/issues/234
+// nolint: lll // ignore long line length due to variable names
+func (c *Client) CallWithHeaders(method, u string, body, v interface{}, headers map[string]string) (*Response, error) {
```

Comment:
Should we update the `u` variable in the function signature to be `url`?

```suggestion
func (c *Client) CallWithHeaders(method, url string, body, v interface{}, headers map[string]string) (*Response, error) {
```

Reply:
should be addressed in new change